pkgs/development/libraries/aws-crt-cpp/default.nix

```diff
@@ -17,30 +17,21 @@
 
 stdenv.mkDerivation rec {
   pname = "aws-crt-cpp";
-  version = "0.17.28";
+  version = "0.17.32";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "awslabs";
     repo = "aws-crt-cpp";
     rev = "v${version}";
-    sha256 = "sha256-4/BgwX8Pa5D2lEn0Dh3JlUiYUtA9u0rWpBixqmv1X/A=";
+    sha256 = "sha256-Za2+7stcdrKou8DlfIIYGu5RYqS2FG5zCITm9pFSWX0=";
   };
 
   patches = [
     # Correct include path for split outputs.
     # https://github.com/awslabs/aws-crt-cpp/pull/325
     ./0001-build-Make-includedir-properly-overrideable.patch
-
-    # Fix build with new input stream api
-    # https://github.com/awslabs/aws-crt-cpp/pull/341
-    # Remove with next release
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done